src/test/scala/com/fulcrumgenomics/commons/CommonsDefTest.scala

```diff
@@ -128,6 +130,51 @@ class CommonsDefTest extends UnitSpec {
     }
   }
 
+  "CommonsDef.maxN" should "return the maximum N elements" in {
+    an[Exception] should be thrownBy Seq.empty[Int].maxN(0)
```

Comment:
If I can have a longestHomopolymer of length 0 in an empty string, why can't I have the top 0 things from an empty seq?